mycore-restapi/src/main/java/org/mycore/restapi/v2/MCRRestDerivates.java

```diff
@@ -417,58 +417,65 @@ public Response patchDerivate(@BeanParam DerivateMetadata der,
 
         LOGGER.debug(der);
         MCRDerivate derivate = MCRMetadataManager.retrieveMCRDerivate(derid);
+        boolean modified = isModified(der, derivate);
+        if (modified) {
+            try {
+                MCRMetadataManager.update(derivate);
+            } catch (MCRAccessException e) {
+                throw MCRErrorResponse.fromStatus(Response.Status.FORBIDDEN.getStatusCode())
+                        .withErrorCode(MCRErrorCodeConstants.MCRDERIVATE_NO_PERMISSION)
+                        .withMessage("You may not update MCRDerivate " + derivate.getId() + ".")
+                        .withDetail(e.getMessage())
+                        .withCause(e)
+                        .toException();
+            }
+        }
+
+        return Response.noContent().build();
+    }
+
+    private static boolean isModified(DerivateMetadata der, MCRDerivate derivate) {
```

Comment:
This method does more than checking for modification - please find a better name
Maybe add the `MCRMetadataManager.update(derivate);` from above and rename to `updateIfModified()`